shell/pages/prefs.vue

```diff
@@ -153,6 +155,13 @@ export default {
     <div class="mt-10">
       <t k="prefs.theme.autoDetail" :pm="pm" :am="am" />
     </div>
+    <hr />
+    <h4 v-t="'prefs.language'" class="mt-10" />
```

Comment:
I'd vote we make this the first entry above `Theme` (if this does move, the `mt-10` should remain)

Reply:
@richard-cox You mean something like this?
<img width="1059" alt="Screenshot 2022-09-22 at 14 27 38" src="https://user-images.githubusercontent.com/18264463/191746971-0d9d3283-3b06-40b5-8eb7-27448cb47042.png">